_schemaV2.graphql

```diff
@@ -2224,46 +2226,44 @@ type ArtistSeries {
     priceRange: String
     saleID: ID
     size: Int
+
+    # Filter results by Artwork sizes
     sizes: [ArtworkSizes]
     sort: String
     tagID: String
     width: String
   ): FilterArtworksConnection
   forSaleArtworksCount: Int!
+
+  # A globally unique ID.
+  id: ID!
   image: Image
-  imageHeight: Int
-  imageURL: String
-  imageWidth: Int
 
-  # Unique ID for this artist series
+  # A type-specific Gravity Mongo Document ID.
   internalID: ID!
   published: Boolean!
   representativeArtworkID: ID
   slug: String!
   title: String!
 }
 
-# The connection type for ArtistSeries.
+# A connection to a list of items.
 type ArtistSeriesConnection {
   # A list of edges.
   edges: [ArtistSeriesEdge]
-
-  # A list of nodes.
-  nodes: [ArtistSeries]
```

Comment:
Not sure why the Gravity connection have `edges` _and_ `nodes`? AFAIK it's always just `edges`